manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -172,7 +172,12 @@ public function setManagerConfig(ManagerConfig $managerConfig): void
      */
     public function registerContainerConfiguration(LoaderInterface $loader): void
     {
-        $configDir = $this->getProjectDir().'/app/config';
+        $configDir = $this->getProjectDir().'/config';
+
+        if (!file_exists($configDir)) {
```

Comment:
What if the `config` directory exists but the `parameters.yml` file is still in `app/config`?

Reply:
yes I think we should